xml-parser/src/main/java/org/uniprot/core/xml/uniref/UniRefEntryLightConverter.java

```diff
@@ -57,9 +57,10 @@ private void updateMemberValuesFromXml(
             UniRefEntryLightBuilder builder, List<MemberType> members) {
         for (MemberType memberType : members) {
             DbReferenceType dbReference = memberType.getDbReference();
-            if (dbReference.getType().equalsIgnoreCase(PROPERTY_UNIPARC_ID)) {
-                builder.membersAdd(dbReference.getId());
-                builder.memberIdTypesAdd(UniRefMemberIdType.UNIPARC);
+            if (dbReference.getType().equalsIgnoreCase(UniRefMemberIdType.UNIPARC.getXmlName())) {
+                UniRefMemberIdType idType = UniRefMemberIdType.UNIPARC;
+                builder.membersAdd(dbReference.getId() + "," + idType.getDisplayOrder());
+                builder.memberIdTypesAdd(idType);
```

Comment:
I didn't follow why we need order separated by a comma.

Reply:
Probably we should put comment here

Reply:
done